airflow/models/mappedoperator.py

```diff
@@ -318,6 +318,7 @@ def get_serialized_fields(cls):
             "dag",
             "deps",
             "is_mapped",
+            "mapped_kwargs",  # This needs to be able to accept XComArg.
```

Comment:
Do you mean
```suggestion
            "mapped_kwargs",  # This is needed to be able to accept XComArg.
```

Reply:
Yeah. The _this_ in my sentence meant _the `mapped_kwargs` attribute_, but in yours it means _the `"mapped_kwargs"` entry here_.